src/status_im/ui/screens/contacts/events.cljs

```diff
@@ -90,11 +91,12 @@
  (fn [{:keys [db] :as cofx} [_ _ contact-identity]]
    (let [current-account (:account/account db)
          fx              {:db (assoc db :contacts/new-identity contact-identity)}]
-     (if (new-chat.db/validate-pub-key contact-identity current-account)
-       fx
-       (handlers-macro/merge-fx cofx
-                                fx
-                                (add-contact-and-open-chat contact-identity))))))
+     (let [validation-result (new-chat.db/validate-pub-key contact-identity current-account)]
```

Comment:
we already have `let` at line 92, lets just put binding there

Reply:
@rasom Sure, done and pushed :+1: